### ISSUE TITLE
remove gratipay from webpage

### ISSUE THREAD

**Issue by tooomm** [badges/shields#1966]
Since the service is discontinued it's good that the badge still shows a relevant information. The webpage however doesn't need to advertise that the badge is still supported or exists at all.

At one point the code supporting that badge can get removed, but for now it should only be removed from the webpage.

Thoughts?

**Comment by PyvesB** [comment]
Yes, I think you're right here, we forgot to remove the deprecated badge from the examples. Would you like to submite a pull request? 😃

**Comment by tooomm** [comment]
Sure, I'm not sure though with the recent changes to the webpage...

Is it this one?
https://github.com/badges/shields/blob/ae190c5f073e828ec2c0bc522f5dcff37d74b7cd/lib/all-badge-examples.js#L809

**Comment by PyvesB** [comment]
Yes, I believe so.
